### DISCUSSION TITLE
'Failed to parse source for import analysis because the content contains invalid JS syntax.' Why is this error being displayed?

### DISCUSSION THREAD

**Discussion by anikghosh256** [nuxt/nuxt#17452] [Questions]
` ERROR  Failed to parse source for import analysis because the content contains invalid JS syntax. You may need to install appropriate plugins to handle the .css file format.`

**Comment by lihanspace** [comment]
```powershell
yarn build
```

```powershell
 ERROR  Failed to parse source for import analysis because the content contains invalid JS syntax. If you are using JSX, make sure to name the file with the .jsx or .tsx extension.
```

This is the first time I have seen this error message.

It doesn't break the build or tell me where the error occurred.

Nuxt CLI v3.0.0-rc.3-27550969.a4a3cff

**Comment by TheAlexLichter** [comment]
Can someone provide a reproduction? 🤔

**Reply by lihanspace** [reply]
My project is a little complicated.
I haven't found out where the mistake is for the time being.
I searched the error information. The error was output by `vite:import-analysis`, but I did not find the wrong import.
The built project can also run normally.

**Reply by lihanspace** [reply]
I found where the error occurred.

```bash
npx nuxi init nuxt3-error

cd nuxt3-error
yarn install

yarn add -D sass
```

```ts [nuxt.config.ts]
// nuxt.config.ts
import { defineNuxtConfig } from 'nuxt'

// https://v3.nuxtjs.org/api/configuration/nuxt.config
export default defineNuxtConfig({
  postcss: {
    plugins: {
      cssnano: {},
    },
  },
})
```

```vue [app.vue]
// app.vue
<template>
  <div>
    <NuxtWelcome />
  </div>
</template>

<style lang="scss" scoped>
.box1 {
  background-image: url("./assets/img/img.png");
  .t0:after {
    content: 'h';
  }
}
</style>
```

```bash
yarn build
```

Then you can see the console output ` ERROR  Failed to parse source for import analysis because the content contains invalid JS syntax. If you are using JSX, make sure to name the file with the .jsx or .tsx extension.` error message.

The path being processed by `vite:import-analysis` is `D:/work/code/web/pc/nuxt3-error/app.vue?vue&type=style&index=0&scoped=true&lang.scss`
